src/commonMain/kotlin/org/hisp/dhis/lib/expression/spi/ValueType.kt

```diff
@@ -84,7 +84,7 @@ enum class ValueType(internal val default: Any) {
     }
 
     companion object {
-        fun allSame(types: List<ValueType>): Boolean {
+        fun allSame(types: Array<ValueType>): Boolean {
```

Comment:
This class (ValueType) is JsExported. In Kotlin 1.9.21, this type `List<ValueType>` was translated to `Any ` in javascript (lists were not supported). In Kotlin 2.0.21, this type is translated to `KtList<ValueType>`. The class `KtList` is an auxiliary class for lists in javascript. This is not native javascript but a class created by Kotlin.